Packs/Respond/Integrations/RespondAnalyst/RespondAnalyst.py

```diff
@@ -1094,32 +1124,36 @@ def main():
             demisto.setLastRun(next_run)
             demisto.incidents(incidents)
 
-        elif demisto.command() == 'respond-close-incident':
+        elif demisto.command() == 'mad-close-incident':
```

Comment:
The integration is called RespondAnalyst, are we sure that we want the prefix to be `mad`?